src/core/function_messages_cqrs.tf

```diff
@@ -85,6 +108,10 @@ module "function_messages_cqrs" {
       "AzureWebJobs.CosmosApiMessageStatusChangeFeedForView.Disabled" = "0"
       "AzureWebJobs.HandleMessageViewUpdateFailures.Disabled"         = "0"
       "AzureWebJobs.UpdateCosmosMessageView.Disabled"                 = "0"
+      "AzureWebJobs.UpdatePaymentOnMessageView.Disabled"              = "0"
+      "AzureWebJobs.HandlePaymentUpdateFailures.Disabled"             = "0"
+      "AzureWebJobs.CosmosApiMessagesChangeFeed.Disabled"             = "0"
+      "AzureWebJobs.HandleMessageChangeFeedPublishFailures.Disabled"  = "0"
```

Comment:
We must remember to manually mark the environament viariable as slot-dependent on the azure developer portal

Reply:
since this function has only async operations probably we can dismantle the staging slot